project/Scriptit.scala

```diff
@@ -53,7 +53,18 @@ object Scriptit {
   }
 
   def scriptitParser(scriptitBase: File, script: String): Parser[Seq[String]] =
-    Defaults.distinctParser(scriptitTests(scriptitBase, script).toSet, raw = false)
+    distinctParser(scriptitTests(scriptitBase, script).toSet, raw = false)
+
+  def distinctParser(exs: Set[String], raw: Boolean): Parser[Seq[String]] =
```

Comment:
1. Could you leave an comment as to what this is and why it's here?

2. Outside of hotfixing this here like this, what are we going to do about it?